manager/apis/app/v1alpha1/m4dstorageaccount_types.go

```diff
@@ -16,8 +16,9 @@ type M4DStorageAccountSpec struct {
 	// Endpoint
 	Endpoint string `json:"endpoint"`
 	// +required
-	// Region
-	Region string `json:"region"`
+	// +kubebuilder:validation:MinItems=1
+	// Regions
+	Regions []string `json:"region"`
```

Comment:
FYI I think that endpoints and regions are usually coupled but ok for now, we can change it later as needed.